pkg/asset/installconfig/gcp/validation.go

```diff
@@ -248,3 +249,24 @@ func ValidateEnabledServices(ctx context.Context, client API, project string) er
 	}
 	return nil
 }
+
+// ValidateProjectRegion determines whether the region is valid for the project
+func validateRegion(client API, ic *types.InstallConfig, fieldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if ic.GCP.ProjectID != "" && ic.GCP.Region != "" {
+		computeRegions, err := client.GetRegions(context.TODO(), ic.GCP.ProjectID)
+		if err != nil {
+			return append(allErrs, field.InternalError(fieldPath.Child("project"), err))
+		} else if len(computeRegions) == 0 {
+			return append(allErrs, field.Invalid(fieldPath.Child("project"), ic.GCP.ProjectID, "no regions found"))
+		}
+
+		validRegions := getValidatedRegions(computeRegions)
+		if _, ok := validRegions[ic.GCP.Region]; !ok {
+			return append(allErrs, field.Invalid(fieldPath.Child("region"), ic.GCP.Region, "invalid region"))
+		}
+	}
```

Comment:
This doesn't look right to me... wouldn't this require that any regions in the install config be in the QE whitelist? When trying to test I hit a different issue.

Reply:
So ignore my earlier comment about the credentials, but it looks like my comment here is correct. This block is preventing specifying europe-west8 in an install config:

```
ERROR failed to fetch Install Config: failed to load asset "Install Config": failed to create install config: platform.gcp.region: Invalid value: "europe-west8": invalid region 
```